lib/shared/models/selection-state.js

```diff
@@ -75,21 +75,54 @@ exports.setImage = (image) => {
 }
 
 /**
- * @summary Get drive
+ * @summary Get all selected drives' devices
  * @function
  * @public
  *
- * @returns {Object} drive
+ * @returns {String[]} selected drives' devices
  *
  * @example
- * const drive = selectionState.getDrive();
+ * for (driveDevice of selectionState.getDevices()) {
+ *   console.log(driveDevice)
+ * }
  */
-exports.getDrive = () => {
-  return _.find(availableDrives.getDrives(), {
-    device: store.getState().getIn([ 'selection', 'drive' ])
+exports.getDevices = () => {
+  return store.getState().getIn([ 'selection', 'drives' ]).toJS()
+}
+
+/**
+ * @summary Get all selected drive objects
+ * @function
+ * @public
+ *
+ * @returns {String[]} selected devices
```

Comment:
Hmm, the `@summary` says `selected drive objects`, but the `@returns` says a String array of `selected devices`??

I see that the old `.getDrive()` function returned a drive Object, which seems a bit inconsistent naming-wise, given that all the other Drive-related functions just accept a String parameter. So perhaps for clarity, it'd be worth having a `.getSelectedDrives()` function which returns an array of Strings, and a `.getSelectedDriveObjects()` function (if it's indeed needed) which returns an array of drive Objects?